lib/plugins/iob.js

```diff
@@ -287,6 +287,21 @@ function init(ctx) {
       , intentHandler: alexaIOBIntentHandler
     }]
   };
+  
+  function googleHomeIOBIntentHandler(result, next, sbx) {
+    var preamble = result && result.parameters && result.parameters.givenName ? result.parameters.givenName + ' has ' : 'You have ';
+    var message = preamble + getIob(sbx) + ' insulin on board';
+    next(message);
+  }
+
+  iob.googleHome = {
+    intentHandlers: [{
+      intent: 'CurrentMetric'
+      , routableSlot: 'metric'
+      , slots: ['iob', 'insulin on board', 'insulin']
+      , intentHandler: googleHomeIOBIntentHandler
+    }]
+  };
```

Comment:
Same here let's aim to restructure both Home and Alexa so the IOB reporting code is in the plugin, not in iob.js

Reply:
Alternatively, would it be possible to add a single method that output a string for speech and reuse that in both Home and Alexa? That'd maybe be the cleanest solution?